database/config/config.js

```diff
@@ -4,12 +4,15 @@ config();
 
 export const development = {
   use_env_variable: 'DATABASE_URL',
+  logging: false,
 };
 
 export const test = {
   use_env_variable: 'TEST_DATABASE_URL',
+  logging: false,
```

Comment:
Why did you disable logging?

Reply:
The logs always show up on the console when sequelize communicates with the database. For instance User.create will cause sequelize to log the associated `sql insert query` to the console. We really dont need to see that since we know queries are being run behind the scenes

Reply:
Sometimes, those logs are helpful for debugging in `development`. I think we can leave that one out

Reply:
Ok then... I'll return it right away and push again.
Thanks.

Reply:
You can disable `testing` and `production`. Just leave `development` for debugging sake